com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 ### Changed
 ### Fixed
+- Fixed a bug that caused visual artefacts when pausing Alembic playback. 
```

Comment:
Suggestion, where we could probably include a slice of the cause, which explicitly involves motion vectors (?):
`Fixed a bug that was causing visual artefacts due to motion vector persistence when pausing Alembic playback.`